runtime/src/snapshot_config.rs

```diff
@@ -75,24 +75,39 @@ impl Default for SnapshotConfig {
 
 impl SnapshotConfig {
     /// A new snapshot config used for only loading at startup
-    #[must_use]
```

Comment:
Removed this because I wanted both `fn new_*` functions to be consistent and it's not necessary to force callers to use the result of a new struct, they will obviously want to use it.